diff -uraN maui-shell-0.6.6/casklib/CMakeLists.txt omv-maui-shell-0.6.6/casklib/CMakeLists.txt
--- maui-shell-0.6.6/casklib/CMakeLists.txt	2023-05-23 18:16:23.000000000 +0200
+++ omv-maui-shell-0.6.6/casklib/CMakeLists.txt	2024-01-10 18:15:12.765938538 +0100
@@ -70,9 +70,9 @@
 
 find_package(Qt5 ${REQUIRED_QT_VERSION} REQUIRED COMPONENTS Core Quick Concurrent DBus Sql)
 find_package(KF5 ${REQUIRED_KF5_VERSION} REQUIRED COMPONENTS I18n Service KIO)
-find_package(MauiKit REQUIRED)
-find_package(MauiKit REQUIRED COMPONENTS FileBrowsing)
-find_package(MauiMan REQUIRED)
+find_package(MauiKit3 REQUIRED)
+find_package(MauiKit3 REQUIRED COMPONENTS FileBrowsing)
+find_package(MauiMan3 REQUIRED)
 find_package(CaskServer REQUIRED)
 
 if(QUICK_COMPILER)
diff -uraN maui-shell-0.6.6/CMakeLists.txt omv-maui-shell-0.6.6/CMakeLists.txt
--- maui-shell-0.6.6/CMakeLists.txt	2023-05-23 18:16:23.000000000 +0200
+++ omv-maui-shell-0.6.6/CMakeLists.txt	2024-01-10 18:14:36.827563854 +0100
@@ -14,8 +14,8 @@
 find_package(Qt5 ${REQUIRED_QT_VERSION} REQUIRED NO_MODULE COMPONENTS Qml Quick Svg QuickControls2 WaylandCompositor)
 find_package(KF5 ${REQUIRED_KF5_VERSION} REQUIRED COMPONENTS I18n Config Service)
 
-find_package(MauiKit)
-find_package(MauiMan)
+find_package(MauiKit3)
+find_package(MauiMan3)
 find_package(CaskServer)
 find_package(MauiCore REQUIRED COMPONENTS Audio Notifications Power)
 
diff -uraN maui-shell-0.6.6/src/CMakeLists.txt omv-maui-shell-0.6.6/src/CMakeLists.txt
--- maui-shell-0.6.6/src/CMakeLists.txt	2023-05-23 18:16:23.000000000 +0200
+++ omv-maui-shell-0.6.6/src/CMakeLists.txt	2024-01-10 18:16:07.376970142 +0100
@@ -26,7 +26,7 @@
     ${cask_ASSETS}
     )
 
-target_link_libraries(${PROJECT_NAME} MauiKit MauiMan CaskLib CaskServer::CaskServerLib Qt5::Qml Qt5::Quick Qt5::QuickControls2 Qt5::Svg KF5::Service Qt5::WaylandCompositor Qt5::DBus KF5::I18n KF5::ConfigCore)
+target_link_libraries(${PROJECT_NAME} MauiKit3 MauiMan3 CaskLib CaskServer::CaskServerLib Qt5::Qml Qt5::Quick Qt5::QuickControls2 Qt5::Svg KF5::Service Qt5::WaylandCompositor Qt5::DBus KF5::I18n KF5::ConfigCore)
 
 install(TARGETS ${PROJECT_NAME} ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 install(DIRECTORY assets/wallpapers/ DESTINATION ${KDE_INSTALL_WALLPAPERDIR}/Cask)
diff -uraN maui-shell-0.6.6/src/main.cpp omv-maui-shell-0.6.6/src/main.cpp
--- maui-shell-0.6.6/src/main.cpp	2023-05-23 18:16:23.000000000 +0200
+++ omv-maui-shell-0.6.6/src/main.cpp	2024-01-10 18:17:37.422476008 +0100
@@ -42,7 +42,7 @@
 #include <sys/wait.h>
 #include <unistd.h>
 
-#include <MauiKit/Core/mauiapp.h>
+#include <MauiKit3/Core/mauiapp.h>
 #include <KI18n/KLocalizedString>
 
 
@@ -60,9 +60,9 @@
 
 #include "../cask_version.h"
 
-#include <MauiMan/screenmanager.h>
-#include <MauiMan/thememanager.h>
-#include <MauiMan/mauimanutils.h>
+#include <MauiMan3/screenmanager.h>
+#include <MauiMan3/thememanager.h>
+#include <MauiMan3/mauimanutils.h>
 
 #define ZPACES_URI "Zpaces"
 #define CASK_URI "org.maui.cask"
diff -up maui-shell-0.6.6/casklib/src/CMakeLists.txt.omv~ maui-shell-0.6.6/casklib/src/CMakeLists.txt
--- maui-shell-0.6.6/casklib/src/CMakeLists.txt.omv~	2024-01-10 23:26:01.464819240 +0100
+++ maui-shell-0.6.6/casklib/src/CMakeLists.txt	2024-01-10 23:26:10.680880287 +0100
@@ -117,12 +117,12 @@ target_include_directories(${PROJECT_NAM
 
 target_link_libraries(${PROJECT_NAME}
     PRIVATE
-    MauiMan::MauiMan
+    MauiMan3::MauiMan3
     CaskServer::CaskServerLib
     MauiCore::Audio
     MauiCore::Notifications
-    MauiKit
-    MauiKit::FileBrowsing
+    MauiKit3
+    MauiKit3::FileBrowsing
     KF5::I18n
     KF5::Service
     KF5::ConfigCore
diff -up maui-shell-0.6.6/casklib/src/code/controllers/db.h.omv~ maui-shell-0.6.6/casklib/src/code/controllers/db.h
--- maui-shell-0.6.6/casklib/src/code/controllers/db.h.omv~	2024-01-10 23:29:42.594233651 +0100
+++ maui-shell-0.6.6/casklib/src/code/controllers/db.h	2024-01-10 23:29:47.458264080 +0100
@@ -26,7 +26,7 @@
 #include <QVariantMap>
 #include <QSqlDatabase>
 
-#include <MauiKit/Core/fmh.h>
+#include <MauiKit3/Core/fmh.h>
 
 #include "fmstatic.h"
 
diff -up maui-shell-0.6.6/casklib/src/code/models/appsmodel.h.omv~ maui-shell-0.6.6/casklib/src/code/models/appsmodel.h
--- maui-shell-0.6.6/casklib/src/code/models/appsmodel.h.omv~	2024-01-10 23:29:53.854304073 +0100
+++ maui-shell-0.6.6/casklib/src/code/models/appsmodel.h	2024-01-10 23:29:56.894323072 +0100
@@ -2,7 +2,7 @@
 #define APPSMODEL_H
 
 #include <QObject>
-#include <MauiKit/Core/mauilist.h>
+#include <MauiKit3/Core/mauilist.h>
 
 class AppsModel : public MauiList
 {
diff -up maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.cpp.omv~ maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.cpp
--- maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.cpp.omv~	2024-01-10 23:30:14.862435264 +0100
+++ maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.cpp	2024-01-10 23:30:19.758465805 +0100
@@ -1,7 +1,7 @@
 #include "recentfilesmodel.h"
 
-#include <MauiKit/FileBrowsing/fileloader.h>
-#include <MauiKit/FileBrowsing/fmstatic.h>
+#include <MauiKit3/FileBrowsing/fileloader.h>
+#include <MauiKit3/FileBrowsing/fmstatic.h>
 
 #include <QFileSystemWatcher>
 #include <QDebug>
diff -up maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.h.omv~ maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.h
--- maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.h.omv~	2024-01-10 23:30:04.362369723 +0100
+++ maui-shell-0.6.6/casklib/src/code/models/recentfilesmodel.h	2024-01-10 23:30:08.014392525 +0100
@@ -1,7 +1,7 @@
 #ifndef RECENTFILESMODEL_H
 #define RECENTFILESMODEL_H
 
-#include <MauiKit/Core/mauilist.h>
+#include <MauiKit3/Core/mauilist.h>
 
 #include <QObject>
 
diff -up maui-shell-0.6.6/src/main.cpp.omv~ maui-shell-0.6.6/src/main.cpp
diff -up maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.cpp.omv~ maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.cpp
--- maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.cpp.omv~	2024-01-10 23:35:51.018298614 +0100
+++ maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.cpp	2024-01-10 23:35:58.166384449 +0100
@@ -1,13 +1,13 @@
 #include "mauimaninterface.h"
 #include <QDBusInterface>
 
-#include <MauiMan/settingsstore.h>
-#include <MauiMan/mauimanutils.h>
+#include <MauiMan3/settingsstore.h>
+#include <MauiMan3/mauimanutils.h>
 
-#include <MauiMan/backgroundmanager.h>
-#include <MauiMan/thememanager.h>
-#include <MauiMan/screenmanager.h>
-#include <MauiMan/inputdevicesmanager.h>
+#include <MauiMan3/backgroundmanager.h>
+#include <MauiMan3/thememanager.h>
+#include <MauiMan3/screenmanager.h>
+#include <MauiMan3/inputdevicesmanager.h>
 
 MauiManInterface::MauiManInterface(QObject *parent) : QObject(parent)
   ,m_background(nullptr)
diff -up maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.h.omv~ maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.h
--- maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.h.omv~	2024-01-10 23:35:43.810211125 +0100
+++ maui-shell-0.6.6/casklib/src/code/controllers/mauimaninterface.h	2024-01-10 23:35:48.430267307 +0100
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <QObject>
-#include <MauiMan/formfactormanager.h>
+#include <MauiMan3/formfactormanager.h>
 
 namespace MauiMan
 {
